calendar-bundle/src/Resources/config/services.yml

```diff
@@ -13,3 +13,14 @@ services:
             - '@security.helper'
         tags:
             - { name: contao.picker_provider, priority: 96 }
+
+    contao_calendar.picker.event_content_provider:
```

Comment:
We have agreed on using the class name as service key for all new services, so this needs to become `Contao\CalendarBundle\Picker\EventContentPickerProvider`.